tests/debugging/probe/test_remoteconfig.py

```diff
@@ -177,3 +185,79 @@ def cb(e, ps):
         }
     finally:
         config.diagnostics_interval = old_interval
+
+
+def test_multiple_configs():
+    events = set()
+
+    def cb(e, ps):
+        events.add((e, frozenset([p.probe_id if isinstance(p, Probe) else p for p in ps])))
```

Comment:
```suggestion
        events.add((e, {p.probe_id if isinstance(p, Probe) else p for p in ps}))
```
there usually is no need for   `frozenset`s in tests.